components/collective-page/sections/Contributions.js

```diff
@@ -59,378 +107,264 @@ const I18nFilters = defineMessages({
-        id
-        collective: member {
+        nodes {
           id
-          name
-          slug
-          type
-          currency
-          isIncognito
-          description
-          imageUrl(height: 128)
-          backgroundImageUrl(height: 200)
-          tags
-          settings
-          parentCollective {
-            id
-            backgroundImageUrl
+          role
+          tier {
             name
-            slug
+            description
           }
-          host {
-            id
+          since
+          totalDonations {
+            currency
+            valueInCents
           }
-          stats {
+          publicMessage
+          description
+          account {
             id
-            backers {
-              id
-              all
+            name
+            slug
+            type
+            isIncognito
+            isAdmin
+            isHost
+            imageUrl
+            backers: members(role: [BACKER]) {
+              totalCount
             }
           }
         }
       }
-      memberOf(onlyActiveCollectives: true, limit: 1500) {
-        id
-        role
-        since
-        description
-        stats {
-          id
-          totalDonations
-        }
-        collective {
+      hostedAccounts: memberOf(role: [HOST], accountType: [COLLECTIVE]) {
+        totalCount
+      }
+      connectedAccounts: members(role: [CONNECTED_ACCOUNT]) {
+        totalCount
+        nodes {
           id
-          name
-          slug
-          type
-          currency
-          isIncognito
-          description
-          imageUrl(height: 128)
-          backgroundImageUrl(height: 200)
-          tags
-          parentCollective {
-            id
-            backgroundImageUrl(height: 200)
-          }
-          host {
-            id
+          role
+          tier {
+            name
+            description
           }
-          stats {
+          publicMessage
+          description
+          account {
             id
-            backers {
-              id
-              all
-            }
+            name
+            slug
+            type
+            isIncognito
+            isAdmin
+            isHost
+            imageUrl
           }
         }
       }
     }
   }
 `;
 
-export const getContributionsSectionQueryVariables = slug => {
-  return { slug };
+const SectionContributions = ({ collective }) => {
+  const intl = useIntl();
+  const [filter, setFilter] = React.useState(FILTERS.ALL);
+  const { data, loading, fetchMore } = useQuery(contributionsSectionQuery, {
+    variables: { slug: collective.slug, limit: PAGE_SIZE, offset: 0, ...FILTER_PROPS[0].where },
+    context: API_V2_CONTEXT,
+  });
+
+  const handleLoadMore = () => {
+    const offset = memberOf.nodes.length;
+    const selectedFilter = FILTER_PROPS.find(f => f.id === filter);
+    fetchMore({
+      variables: { offset, ...selectedFilter.where },
+      updateQuery: (prev, { fetchMoreResult }) => {
```

Comment:
Using `updateQuery` with `fetchMore` is deprecated:

> The updateQuery callback for fetchMore is deprecated, and will be removed
> in the next major version of Apollo Client.
> 
> Please convert updateQuery functions to field policies with appropriate
> read and merge functions, or use/adapt a helper function (such as
> concatPagination, offsetLimitPagination, or relayStylePagination) from
> @apollo/client/utilities.
> 
> The field policy system handles pagination more effectively than a
> hand-written updateQuery function, and you only need to define the policy
> once, rather than every time you call fetchMore.